components/image/image.css

```diff
@@ -0,0 +1,20 @@
+@import url('/index.css');
+
+
+.image-container {
+    width: 60%;
+    border: 2px solid gray;
+    border-radius: 5px;
+    background-color: var(--bg-color);
+}
+
+.image {
+    max-inline-size: 100%;
+    block-size: auto;
+}
+
+.round {
+    border-radius: 50%;
+    width: 15rem;
+    height: 15rem;
+}
```

Comment:
stick to either rem or px.
using rem is best practice.

Reply:
i used rem everywhere but for borders i used px only because px is best practice for borders